snapshot/snapshot.go

```diff
@@ -116,6 +120,12 @@ func AsynchronousRemove(config *SnapshotterConfig) error {
 	return nil
 }
 
+// MinLayerSize sets the smallest layer that will be mounted remotely.
+func MinLayerSize(config *SnapshotterConfig, minLayerSize int) error {
```

Comment:
This should really be a function that returns `Opt`.

```
func WithMinLayerSize(minLayerSize int) Opt {
      return func(c *SnapshotterConfig) error {
               c.minLayerSize = minLayerSize
               return nil
      }
}
```

Reply:
I discussed this with Kern. This function follows the model of AsynchronousRemove, which directly manipulates the config instead of returning a function that manipulates the config. This model is distinct from the WithFoo model used elsewhere in the project for options.
Making this change would require also changing AsynchronousRemove, and the handler that calls them.

Reply:
`AsynchronousRemove` is already an `Opt`. Personally, I think `AsynchronousRemove` should be renamed to `WithAsynchronousRemove` and we should use @rdpsin's suggestion here.